explore-htap.md

```diff
@@ -31,6 +31,24 @@ The following are the typical use cases of HTAP:
 
 For more information about use cases of TiDB HTAP, see [blogs about HTAP on the PingCAP website](https://en.pingcap.com/blog/?tag=htap).
 
+To enhance the overall performance of TiDB, it is recommended to use HTAP in the following technical scenarios:
+
+- Improve analytical processing performance
+
+    If your application involves complex analytical queries, such as aggregation and join operations, and these queries are performed on a large amount of data (more than 10 million rows), the row-based storage engine [TiKV](/tikv-overview.md) might not meet your performance requirements if the tables in these queries cannot effectively use indexes or have low index selectivity.
+
+- Hybrid workload isolation
+
+   While dealing with high-concurrency Online Transactional Processing (OLTP) workloads, your system might also need to handle some OLAP workloads. To ensure the overall system stability, you expect to avoid the impact of OLAP queries on OLTP performance.
+
+- Simplify the ETL technology stack
+
+    When the amount of data to be processed is of medium scale (less than 100 TB), the data processing and scheduling processes are relatively simple, and the concurrency is not high (less than 10), you might want to simplify the technology stack of your system. By replacing multiple different technology stacks used in OLTP, ETL, and OLAP systems with a single database, you can meet the requirements of both transactional systems and analytical systems. This reduces technical complexity and the need for maintenance personnel.
+
+- Strongly consistent analysis
+
+    To achieve real-time, strongly consistent data analysis and calculation, and ensure the analysis results to be completely consistent with the data, you need to avoid data latency and inconsistency issues.
```

Comment:
```suggestion
    To achieve real-time, strongly consistent data analysis and calculation, and ensure the analysis results to be completely consistent with the transactional data, you need to avoid data latency and inconsistency issues.
```